src/components/spreadsheet/table-wrapper.js

```diff
@@ -847,6 +1131,11 @@ const TableWrapper = (props) => {
                     EQUIPMENT_TYPES.GENERATOR
                 ) {
                     updateGeneratorCells(params);
+                } else if (
+                    params.data.metadata?.equipmentType ===
```

Comment:
Is optional chaining really necessary here ?

Reply:
not really